realm/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -327,6 +335,15 @@ public Builder deleteRealmIfMigrationNeeded() {
         }
 
         /**
+         * Settings this will create a in-memory Realm instance.
```

Comment:
Change this something like: 
"Setting this will create an in-memory Realm instead of of saving it to disk. In-memory Realms might still use disk space if memory is running low, but all files created by a in-memory Realm will be deleted when the Realm is closed."